File: Financial_Webservice/Financial_Webservice/Models/UserDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Financial_Webservice.Models
{
    public class UserDto
    {
        public Guid _id { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        public string email { get; set; }
        public string phone { get; set; }
        public string userName { get; set; }
        public string password { get; set; }
        public string token { get; set; }
    }
}

File: FinacialManager_iOS/FinacialManager_iOS/Classes/Models/UserDto/UserDto.m
//
//  UserDto.m
//  FinacialManager_iOS
//
//  Created by ThanhSon on 3/17/18.
//  Copyright © 2018 ThanhSon. All rights reserved.
//

#import "UserDto.h"

@implementation UserDto

-(id)initWithData:(NSDictionary *)dic {
    self = [super init];
    if (dic) {
        if ([dic isKindOfClass:[NSDictionary class]]) {
            IO(_id);
            IO(firstName);
            IO(lastName);
            IO(email);
            IO(phone);
            IO(userName);
            IO(password);
        }
    }
    return self;
}

- (NSMutableDictionary *)getJSONObject {
    NSMutableDictionary *dic = [[NSMutableDictionary alloc] init];
    JO(_id);
    JO(firstName);
    JO(lastName);
    JO(email);
    JO(phone);
    JO(userName);
    JO(password);
    return dic;
}

- (id )getJSONObjectWithMethod:(NSInteger)method{
    NSMutableDictionary *dic = [[NSMutableDictionary alloc] init];
    if (method == METHOD_POST) {
        JO(userName);
        JO(password);
    } else if (method == METHOD_POST_2) {
        JO(userName);
        JO(password);
    }
    return dic;
}

@end
